library_registry/admin/routes.py

```diff
@@ -10,59 +14,91 @@
     'admin', __name__,
     template_folder='templates')
 
+# Using an `after_request` callback, we refresh any token that is within 30
+# minutes of expiring. Change the timedeltas to match the needs of your application.
+
+
+@admin.after_request
+def refresh_expiring_jwts(response):
+    try:
+        exp_timestamp = get_jwt()["exp"]
+        now = datetime.now(timezone.utc)
+        target_timestamp = datetime.timestamp(now + timedelta(minutes=30))
```

Comment:
how long is the default lifespan for a token? I think it's ~10 hours? If it's shorter than that we should make this correspondingly shorter (imo).

Reply:
When we were using tokens, the access token was 1 hour and the refresh token was 30 days. Now that we're supporting cookies, we've had conversations about the best expiration time but haven't nailed down an exact one. It seems like we're somewhere between 8 and 12 hours and should be easy enough to update that when we have it.

Reply:
We can set up our preferred JWT expiration times in the application factory set-up in app.py.  This function is checking access_cookies expiring in 30 minutes and sends a new one.  We still need clarification on how long the access cookies need to exist for and if we want to change the refresh route to update with a smaller or wider window.

Reply:
Let's leave it as is for now. I don't think it's a huge issue and it seems like it will be easy to adjust in the future!